uiowa/uiowa01
# Feature: Collegiate theme lb editor fixes

This feature addresses issue #196. 

# How to Test

1. Run drush @theming.local si collegiate.
2. Create a basic page.
3. Edit the layout.
4. Verify that width of the top tabs, help text, save layout buttons, and vertical tabs at the bottom of the edit screen reflect the container theme settings.
5. Add new layout section (please do not test on one column flex layout #240).
6. Adjust each section to full-width, container, or edge to edge to make sure that each section correctly adjusts and collapses when the browser shrinks down to a smaller breakpoint.
